mubench.reviewsite/tests/TagControllerTest.php

```diff
@@ -64,4 +65,18 @@ function test_add_unknown_tag()
 
         self::assertEquals('test-tag', $misuseTags->get(0)->name);
     }
+
+    function update_tag()
+    {
+        $this->tagController->addTagToMisuse(1, 'test-tag');
+
+        $misuseTags = Misuse::find(1)->misuse_tags;
+        $tag = Tag::where('name', 'test-tag')->first();
+        $this->tagController->updateTag($tag->id, ['name' => 'new-name', 'color' => '#555555']);
+
+        $updatedTag = Tag::where('name', 'test-tag')->first();
+
+        self::assertEquals('new-name', $updatedTag->name);
+        self::assertEquals('#555555', $updatedTag->color);
+    }
 }
```

Comment:
Why no test for delete tag? This should ensure that the tag is also removed from any misuse it is linked to.